rest-api-spec/src/main/resources/rest-api-spec/api/security.enroll_client.json

```diff
@@ -0,0 +1,24 @@
+{
+  "security.enroll_client":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/master/security-aoi-enroll-client.html",
+      "description":"Allows a client to configure itself to communicate with a secured elasticsearch cluster."
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_security/enroll_client",
+          "methods":[
+            "POST"
+          ]
+        }
+      ]
+    }
```

Comment:
The API description requires a `"body": {"description": "..."}` section to signal to clients that a body is required.